### PR TITLE
[15.0][MIG] account_invoice_margin_sale_delivered_sync: Migration to version 15.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT38304

### CODE REVIEW

**Comment by ernestotejeda** [comment]
ping @chienandalu @sergio-teruel

File: account_invoice_margin_sale_delivered_sync/models/sale.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
-    def _prepare_invoice_line(self):
+    def _prepare_invoice_line(self, **optional_values):
         vals = super()._prepare_invoice_line()
```
Comment (sergio-teruel) [inline account_invoice_margin_sale_delivered_sync/models/sale.py]: You are breaking the method.. you must pass optional_values

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

File: account_invoice_margin_sale_delivered_sync/models/sale.py (hunk 1 above)
Comment (ernestotejeda) [reply account_invoice_margin_sale_delivered_sync/models/sale.py]: Right!, fixed.

**Review by sergio-teruel — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

**Pull request merged**